trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/tail_horizontal.py

```diff
@@ -0,0 +1,49 @@
+## @ingroup Methods-Weights-Correlations-General_Aviation
+# tail_horizontal.py
+#
+# Created:  Mar 2016, M Vegh
+# Modified: 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+from SUAVE.Core import Units
+import numpy as np
+
+# ----------------------------------------------------------------------
+#   Tail Horizontal
+# ----------------------------------------------------------------------
+## @ingroup Methods-Weights-Correlations-General_Aviation
+def tail_horizontal(S_h, AR_h, sweep_h, q_c, taper_h, t_c_h,Nult,TOW):       
+    """ weight = SUAVE.Methods.Weights.Correlations.Tube_Wing.tail_horizontal(b_h,sweep_h,Nult,S_h,TOW,mac_w,mac_h,l_w2h,t_c_h)
```

Comment:
Update this docstring

Reply:
done

Reply:
This still lists Tube_Wing. I don't think this is accurate?